ledger/sandbox-on-x/BUILD.bazel

```
@@ -467,6 +467,27 @@ server_conformance_test(
     ],
 )
 
+# By default, participants are tuned for performance. The buffers and caches used by the participant
+# are by default so large that they are not filled by the small amount of data produced by the conformance test.
+# We run one conformance test with small buffer/cache sizes to make sure we cover cases where data doesn't fit
+# into a cache or where multiple buffers have to be combined.
+server_conformance_test(
+    name = "conformance-test-tiny-buffers",
+    server_args = [
+        "--contract-id-seeding=testing-weak",
+        "--participant participant-id=example,port=6865",
+        "--enable-user-management=true",
+        "--events-page-size=2",
+        "--acs-id-page-size=2",
+        "--user-management-max-cache-size=2",
```

Let's downsize the mutable contract state caches as well
```suggestion
        "--user-management-max-cache-size=2",
        "--contract-state-cache-max-size=2",
        "--contract-key-state-cache-max-size=2",
```